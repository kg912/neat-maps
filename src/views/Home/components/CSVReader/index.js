import React, { Component } from 'react';
import ReactCSVReader from 'react-csv-reader';


export default class CSVReader extends Component {
	constructor() {
		super();
	}

	render() {
		const { onCSVLoad } = this.props;
		return (
			<ReactCSVReader
				label=""
				onFileLoaded={onCSVLoad}
			/>
		);
	}
}