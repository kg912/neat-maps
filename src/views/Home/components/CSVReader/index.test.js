import React from 'react';
import { create } from 'react-test-renderer';
import CSVReader from '../CSVReader';

test("CSVReader Snapshot", () => {
	const c = create(<CSVReader onCSVLoad={() => true} />);
	expect(c.toJSON()).toMatchSnapshot();
});