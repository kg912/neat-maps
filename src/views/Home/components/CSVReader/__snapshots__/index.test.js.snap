// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CSVReader Snapshot 1`] = `
<div
  className="csv-reader-input"
>
  
  <input
    accept="text/csv"
    className="csv-input"
    id={null}
    onChange={[Function]}
    style={Object {}}
    type="file"
  />
</div>
`;
